extensions/mssql/src/objectManagement/interfaces.ts

```diff
@@ -443,6 +451,17 @@ export interface Database extends ObjectManagement.SqlObject {
 	azureServiceLevelObjective?: string;
 	azureEdition?: string;
 	azureMaxSize?: string;
+	autoCreateIncrementalStatistics: boolean;
+	autoCreateStatistics: boolean;
+	autoShrink: boolean;
+	autoUpdateStatistics: boolean;
+	autoUpdateStatisticsAsynchronously: boolean;
+	isLedgerDatabase?: boolean;
+	pageVerify?: string;
+	targetRecoveryTimeInSec?: number;
+	databaseReadOnly?: boolean;
+	encryptionEnabled: boolean;
+	userAccess?: string;
```

Comment:
what does user access refer to? should it be restrictAccess?

Reply:
yeah the field name we display is as restricAccess, but we get the smo object name as userAccess, so i tried to match with the name but displaying how SSMS displays it. I can change it to RestricAccess too!